scripts/qa_crowbarsetup.sh

```diff
@@ -3007,6 +3008,11 @@ function oncontroller_testsetup()
         if iscloudver 5plus; then
             /usr/bin/tempest-cleanup --init-saved-state || :
         fi
+        if [[ "$defcore_url" ]]; then
+           wget --no-check-certificate -Odefcore.txt "$defcore_url"
+           # this will overwrite the default temperstoptions
+           $tempestoptions="-- --load-list defcore.txt"
```

Comment:
you need to indent the block 1 more space. also you have a typo in the comment.

why do we need to overwrite here and not append?